mujoco_viewer/mujoco_viewer.py

```diff
@@ -137,6 +139,16 @@ def _key_callback(self, window, key, scancode, action, mods):
         elif key == glfw.KEY_M:
             self._com = not self._com
             self.vopt.flags[mujoco.mjtVisFlag.mjVIS_COM] = self._com
+        # Shadow Rendering
+        elif key == glfw.KEY_O:
+            self._shadows = not self._shadows
+            self.scn.flags[mujoco.mjtRndFlag.mjRND_SHADOW] = self._shadows
+        # Convex-Hull rendering
+        elif key == glfw.KEY_V:
+            self.vopt.flags[
+                mujoco.mjtVisFlag.mjVIS_CONVEXHULL
+            ] = self._convex_hull_rendering
+            self._convex_hull_rendering = not self._convex_hull_rendering
```

Comment:
This line should come before setting the vopt flags

Reply:
This is the correct order:
```
            self._convex_hull_rendering = not self._convex_hull_rendering
            self.vopt.flags[
                mujoco.mjtVisFlag.mjVIS_CONVEXHULL
            ] = self._convex_hull_rendering
```

Reply:
updated: [`23c7cea`](https://github.com/rohanpsingh/mujoco-python-viewer/commit/23c7cea70f9095804301216c39dbadef9866e413)